Modules/Scripted/VPAWVisualize/VPAWVisualize.py

```diff
@@ -371,50 +495,97 @@ def setDefaultParameters(self, parameterNode):
         if not parameterNode.GetParameter("Invert"):
             parameterNode.SetParameter("Invert", "false")
 
-    def process(
-        self, inputVolume, outputVolume, imageThreshold, invert=False, showResult=True
-    ):
+    def process(self, dataDirectory, patientPrefix):
```

Comment:
"process" is an odd name for this function. And the docstring "Run the processing algorithm" is also odd.

considering that the purpose of this function is to return a list of files ordered my modification time

the name and docstring should be adjusted to better reflect what it does

Reply:
I renamed it to `find_and_sort_files_with_prefix` and gave it a meaningful docstring.